src/plugin/Plugin.h

```diff
@@ -130,10 +130,12 @@ class Configuration
 #endif
 	inline Configuration() __attribute__((always_inline))
 		{
-// Only bake in a ZEEK_PLUGIN_ZEEK_VERSION reference into external
-// plugins. The internal ones are in the same binary so the runtime
-// link check shouldn't be needed and we can avoid ccache busting.
-#if defined(ZEEK_PLUGIN_INTERNAL_BUILD) && ! ZEEK_PLUGIN_INTERNAL_BUILD
+// Only bake in a ZEEK_PLUGIN_ZEEK_VERSION reference into external plugins. The
+// internal ones are in the same binary so the runtime link check shouldn't be
+// needed and we can avoid ccache busting. The define gets set in the
+// CMakeLists.txt file for the Zeek::Internal target, which only exists when
+// building Zeek itself.
+#ifndef ZEEK_PLUGIN_SKIP_VERSION_CHECK
```

Comment:
suspect line 17 needs this change as well? CI isn't quite happy either :-(

https://github.com/zeek/zeek/pull/3029/files#diff-8cb93601057dda9faeaaa75290dd6153f18ab6c12e6a2653e399e764f4fe8600R17

Reply:
This would be a btest that works on Linux tested with Zeek 5.2.1:

https://github.com/zeek/zeek/commit/46f33182357a154138e2cf97c85e9c3d38945558

It does use `nm` to find the undefined zeek_version symbol, so it's a pretty low level test, but given we lost the feature possibly useful still?